### PR TITLE
Config update

### PR DESCRIPTION
**Pull request by jas-mckin**
Update the readthedocs configuration. This includes adding a .readthedocs.yaml file, and making some additions to the config.py file. sphinx-tabs and myst_parser packages were added to the requirements.txt file.
sphinx-tabs adds tabs functionality, as described at https://sphinx-tabs.readthedocs.io/en/latest/
myst_parser in requirements.txt permits the sphinx theme to continue to be able to display .md files in addition to .rst files under the new config.

### CODE REVIEW

File: faq/pathogen-subs-guide.rst [hunk 1]
```diff
@@ -263,58 +263,60 @@ for permitted values.
...
+         chr03   3 circular-Chromosome plasmid
 
-Eukaryota
----------
+   .. tab:: Eukaryota
 
-By default eukaryotic chromosomes will be assumed to reside in the nucleus. By default the chromosome topology
-will be assumed to be linear, but it may also be specified.
+      By default eukaryotic chromosomes will be assumed to reside in the nucleus. By default the chromosome topology
+      will be assumed to be linear, but it may also be specified.
```
Comment (carlacummins) [inline faq/pathogen-subs-guide.rst:311]: Why was it **`TOPOLOGY`** in the bacteria tab, but just `topology` here?

File: faq/pathogen-subs-guide.rst [hunk 2]
```diff
@@ -263,58 +263,60 @@ for permitted values.
...
+Note that ‘chromosome’ should here be understood as a general term for a range of complete replicons, including chromosomes of eukaryotes, prokaryotes, 
+and viruses, as well as organellar chromosomes and plasmids. All of these may be submitted within the same chromosome-level assembly.
+
+If your assembly is not complete, you can submit a **contig** or **scaffold** level assembly. The `Assembly levels <https://ena-docs.readthedocs.io/en/latest/submit/assembly.html#assembly-levels>`_ tutorial page explains the assembly levels recognised by ENA.
+
 **chromosome_list.txt** examples
 
-Viruses
---------
+.. tabs::
```
Comment (carlacummins) [inline faq/pathogen-subs-guide.rst:276]: These tabs look great! 🙂

File: faq/pathogen-subs-guide.rst [hunk 3]
```diff
@@ -263,58 +263,60 @@ for permitted values.
...
-Please refer to the `chromosome list file guide <https://ena-docs.readthedocs.io/en/latest/submit/fileprep/assembly.html#chromosome-list-file>`_
+The **chromosome list file** must be provided when the submission contains assembled chromosomes. This is a tab separated file up to four columns. Each row describes each replicon unit within the assembly. Please refer to the `chromosome list file guide <https://ena-docs.readthedocs.io/en/latest/submit/fileprep/assembly.html#chromosome-list-file>`_
 for permitted values.
 
+Note that ‘chromosome’ should here be understood as a general term for a range of complete replicons, including chromosomes of eukaryotes, prokaryotes, 
+and viruses, as well as organellar chromosomes and plasmids. All of these may be submitted within the same chromosome-level assembly.
+
+If your assembly is not complete, you can submit a **contig** or **scaffold** level assembly. The `Assembly levels <https://ena-docs.readthedocs.io/en/latest/submit/assembly.html#assembly-levels>`_ tutorial page explains the assembly levels recognised by ENA.
+
 **chromosome_list.txt** examples
```
Comment (carlacummins) [inline faq/pathogen-subs-guide.rst:274]: Can this be more like a small subheading? Would apply to `manifest.txt example` above too. They get a bit lost on the page.

**Review by carlacummins — APPROVED** [review]
Really good! Minor suggestions

File: faq/pathogen-subs-guide.rst (hunk 1 above)
Comment (jas-mckin) [reply faq/pathogen-subs-guide.rst:309]: this has been corrected to be consistent

File: faq/pathogen-subs-guide.rst (hunk 3 above)
Comment (jas-mckin) [reply faq/pathogen-subs-guide.rst]: The assembly submissions text has been reviewed and re-organised to make more sense

File: faq/pathogen-subs-guide.rst [hunk 4]
```diff
@@ -318,12 +318,18 @@ If your assembly is not complete, you can submit a **contig** or **scaffold** le
          chr04   4 Linear-Chromosome
          chrMi   MIT Linear-Chromosome Mitochondrion
 
+If there are sequences that are associated with a specific chromosome, but order and orientation is unknowm, you can also add sn
```
Comment (carlacummins) [inline faq/pathogen-subs-guide.rst:321]:
```suggestion
If there are sequences that are associated with a specific chromosome, but order and orientation is unknowm, you can also add an
```

File: faq/pathogen-subs-guide.rst [hunk 5]
```diff
@@ -209,23 +210,33 @@ the library description. This can be, for example, the name and/or URL to a spec
...
-   For submission of targeted sequences, please refer to the `targeted sequence submissions guide <https://ena-docs.readthedocs.io/en/latest/submit/sequence.html#how-to-submit-targeted-sequences>`_.
+   **A note on assembly levels**
+
+   This guide includes chromosome list file examples which are used for a **chromosome** level assembly. Note that
+   ‘chromosome’ should here be understood as a general term for a range of complete replicons, including chromosomes of
+   eukaryotes, prokaryotes, and viruses, as well as organellar chromosomes and plasmids. All of these may be submitted
+   within the same chromosome-level assembly.
+
+   If your assembly is not completed, you can submit a **contig** or **scaffold**
+   level assembly. Please refer to the explainer about Assembly levels `here <https://ena-docs.readthedocs.io/en/latest/submit/assembly.html#assembly-levels>`_.
```
Comment (carlacummins) [inline faq/pathogen-subs-guide.rst:230]:
```suggestion
   level assembly. Please refer to the explainer about assembly levels `here <https://ena-docs.readthedocs.io/en/latest/submit/assembly.html#assembly-levels>`_.
```

File: faq/pathogen-subs-guide.rst [hunk 6]
```diff
@@ -209,23 +210,33 @@ the library description. This can be, for example, the name and/or URL to a spec
...
+For submission of targeted sequences, please refer to the `targeted sequence submissions guide <https://ena-docs.readthedocs.io/en/latest/submit/sequence.html#how-to-submit-targeted-sequences>`_.
 
-The instructions here provide tips for submitting microbial pathogen assemblies. For genome assembly submission,
-Webin-CLI (command line interface) needs to be used. The guide for downloading and using Webin-CLI is `here <https://ena-docs.readthedocs.io/en/latest/submit/general-guide/webin-cli.html#webin-cli-submission>`_.
+For genome assembly submission, Webin-CLI (command line interface) needs to be used. The guide for downloading
+and using Webin-CLI is `here <https://ena-docs.readthedocs.io/en/latest/submit/general-guide/webin-cli.html#webin-cli-submission>`_.
 
 .. tip::
-   For submission of targeted sequences, please refer to the `targeted sequence submissions guide <https://ena-docs.readthedocs.io/en/latest/submit/sequence.html#how-to-submit-targeted-sequences>`_.
+   **A note on assembly levels**
```
Comment (carlacummins) [inline faq/pathogen-subs-guide.rst:222]: I think this:
```
.. tip:: A note on assembly levels
    This guide ........ etc

``` 
would result in the tip box having a custom heading (https://sphinxawesome.xyz/demo/notes/). It's absolutely fine as it is, but might be nice to see how it looks?

**Review by carlacummins — APPROVED** [review]

**Review by carlacummins — APPROVED** [review]
Happy for you to merge now 🙂

**Pull request merged**